src/collectFileNames.test.ts

```diff
@@ -0,0 +1,34 @@
+import path from "node:path";
+import { describe, expect, it } from "vitest";
+
+import { collectFileNames } from "./collectFileNames.js";
+
+describe("collectFileNames", () => {
+	it("should collect files with wildcard", async () => {
+		const cwd = path.resolve(import.meta.dirname, "..");
+		const fileNames = await collectFileNames(
+			path.resolve(import.meta.dirname),
+			["*"],
+		);
+		const names = Array.isArray(fileNames)
+			? (fileNames as string[]).map((item) => item.replace(cwd, "<rootDir>"))
+			: undefined;
+
+		// Assert
```

Comment:
[Style] Heh, explicitly adding `// Arrange` + `// Act` + `// Assert` comments is an old practice I used to do. I've mostly dropped it. No preference from me on whether you keep to it, but if you do the point is to explicitly label each section of the test. These only have 1 of 3 labels.